google/cloud/bigquery/storage/__init__.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from google.cloud.bigquery.storage_v1.services.big_query_read.async_client import (
+    BigQueryReadAsyncClient,
+)
+from google.cloud.bigquery.storage_v1.services.big_query_read.client import (
+    BigQueryReadClient,
```

Comment:
Update: I see that we still have the manual wrapper in the `google.cloud.bigquery_storage_v1` package. That's probably the right call. It can hopefully avoid some breaking changes that would affect `google-cloud-bigquery` and `pandas-gbq`.

Still might make sense to import that manual wrapper here to avoid confusion. That is, I'd like the the non-wrapped version to only be available when importing from google.cloud.bigquery.storage_v1.services directly, similarly to how the current version it's only available when importing from the gapic dir.

Reply:
Since there are going to be breaking changes anyway, let's consolidate on `google.cloud.bigquery.storage`

Reply:
Done, `google.cloud.bigquery.storage` is now the main namespace users should use.